app/models/user.rb

```diff
@@ -218,10 +218,9 @@ def student_team
 
   private
 
-  # Ensures that the location column either contains non-whitespace text, or is NULL
-  # This ensures that sorting by location yields useful results
-  def sanitize_location
-    self.location = nil if self.location.blank?
+  def format_location
+    self.location = nil if self.location.blank? # for sorting
+    self.location = self.location.downcase.titlecase unless location.nil?
```

Comment:
😍 `nil.to_s.presence` 
I prefer to add `downcase` to `titlecase` because of this:

` " AMsterdam ".strip.downcase.titlecase # => "Amsterdam" ` 

` " AMsterdam ".strip.titlecase #=> "A Msterdam" `

It was implicitly tested in the specs; I added a more explicit test.